ansible/roles/pmm2-images/vars/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-pmm2_server_repo: release
-pmm_client_repos: "original testing"
+pmm2_server_repo: experimental
```

Comment:
are you sure that we should replace `release` with `experimental`?

Reply:
you are right, I don't think this one is really needed, since it is being https://github.com/Percona-Lab/percona-images/blob/master/packer/pmm2.json#L76 we already changed it to experimental for OVF and AMI hence we can keep this back to default release.

Reply:
others also should be passed as variables. And we need to parametrize them in the job, as well as image name should be set according to the repo.